hologram_dashboard/front-end/static/js/popup.js

```diff
@@ -39,23 +40,23 @@ window.onload = () => {
 
 function populate_info_buttons (item, index) {
```

Comment:
Add high-level comments for each function. Also explain each parameter if it's not obvious.